prime-router/metadata/hl7_mapping/common/datatype/xtn-telecom.yml

```diff
@@ -14,12 +14,12 @@ elements:
         temp: ORN
 
   - name: xtn-use-code-from-system
-    condition: '%resource.system = "pager"'
+    condition: '%resource.system = "pager" and %resource.use.exists().not()'
     value: [ '"BPN"' ]
     hl7Spec: [ '%{hl7TelecomPath}-2' ]
 
   - name: xtn-use-code-from-system
-    condition: '%resource.system = "email"'
```

Comment:
On second look, I don't think we should be changing this condition. `system` of `email` should be enough to use `NET`, especially since `use` might only be set on the internal FHIR bundle to help distinguish between PID.13 and PID.14.

Reply:
It's been a while, but I think this was actually what was causing us to overwrite the field for ELIMS

Reply:
I'll leave that up to @mkalish

Reply:
I'm okay leaving this as is, and we'll open a new story to establish desired behavior for XTNs on those kinds of cases.